truss/templates/server/common/truss_server.py

```diff
@@ -83,6 +90,23 @@ class BasetenEndpoints:
 
     def __init__(self, model: ModelWrapper) -> None:
         self._model = model
+    # Create a Resource object to set the service name
+        resource = Resource(attributes={
+            ResourceAttributes.SERVICE_NAME: self._model._config.get("model_name").split("-")[0]
+        })
+
+        # Create a TracerProvider
+        trace_provider = TracerProvider(resource=resource)
+        # THIS DOES NOT WORK, WE NEED TO FIND A WAY TO LET THE USER DEFINE IT.
```

Comment:
Can you explain more?
What information needs to be user defined?

Reply:
where to export the data, it needs to pass a span processsor.